src/Installers/Windows/SharedFramework/SharedFramework.wixproj

```diff
@@ -84,15 +84,22 @@
   <Target Name="CreateSharedFrameworkNugetPackage" AfterTargets="CopyToArtifactsDirectory;Build">
     <PropertyGroup>
       <MsiFullPath>$(InstallersOutputPath)$(PackageFileName)</MsiFullPath>
+      <CabFullPath>$(InstallersOutputPath)$(Cabinet)</CabFullPath>
```

Comment:
Believe this is incorrect because the CAB file is now embedded. Will let the validation build confirm.